### PR TITLE
Update stackup.py

### PR DESCRIPTION
**Pull request by PipKat**
Last PY file in edb_core folder--Best that makes whoever is reviewing these PRs happy!

### CODE REVIEW

File: pyaedt/edb_core/stackup.py [hunk 1]
```diff
@@ -60,16 +60,12 @@ def stackup_methods(self):
 
     @property
     def stackup_layers(self):
-        """List all stackup layers.
-        
-        Parameters
-        ----------
-
+        """Retrive all stackup layers.
```
Comment (PipKat) [inline pyaedt/edb_core/stackup.py:63]:
```suggestion
        """Retrieve all stackup layers.
```

File: pyaedt/edb_core/stackup.py [hunk 2]
```diff
@@ -78,29 +74,31 @@ def stackup_layers(self):
...
-
         """
         return self.stackup_layers.signal_layers
 
 
-
     @property
     def materials(self):
-        """:return: Dictionary of Materials"""
+        """Retrive all materials.
```
Comment (PipKat) [inline pyaedt/edb_core/stackup.py:89]:
```suggestion
        """Retrieve all materials.
```

File: pyaedt/edb_core/stackup.py [hunk 3]
```diff
@@ -231,16 +225,17 @@ def _add_dielectric_material_model(self, name, material_model):
...
     @aedt_exception_handler
     def stackup_limits(self, only_metals=False):
-        """
+        """Retrieve stackup limits.
 
         Parameters
         ----------
         only_metals : bool, optional
-            The default is ``False``.
+            Whether to retrive only metals. The default is ``False``.
```
Comment (PipKat) [inline pyaedt/edb_core/stackup.py:233]:
```suggestion
            Whether to retrieve only metals. The default is ``False``.
```

File: pyaedt/edb_core/stackup.py [hunk 4]
```diff
@@ -60,16 +60,12 @@ def stackup_methods(self):
 
     @property
     def stackup_layers(self):
-        """List all stackup layers.
-        
-        Parameters
-        ----------
-
+        """Retrieve all stackup layers.
```
Comment (akaszynski) [inline pyaedt/edb_core/stackup.py:63]:
```suggestion
        """Dictionary of all the stackup layers.
```

File: pyaedt/edb_core/stackup.py [hunk 5]
```diff
@@ -78,29 +74,31 @@ def stackup_layers(self):
 
     @property
     def signal_layers(self):
-        """
-
-        Parameters
-        ----------
-
+        """Retrieve all signal layers.
```
Comment (akaszynski) [inline pyaedt/edb_core/stackup.py:77]:
```suggestion
        """List of all signal layers.
```

**Review by akaszynski — APPROVED** [review]
See my notes regarding verb/noun of properties and the line spacing between methods.  Feel free to message me on teams if either doesn't make sense.

File: pyaedt/edb_core/stackup.py [hunk 6]
```diff
@@ -78,29 +74,31 @@ def stackup_layers(self):
...
-
         """
         return self.stackup_layers.signal_layers
 
 
-
     @property
     def materials(self):
-        """:return: Dictionary of Materials"""
+        """Retrieve all materials.
```
Comment (PipKat) [inline pyaedt/edb_core/stackup.py:89]:
```suggestion
        """Dictionary of materials.
```

File: pyaedt/edb_core/stackup.py [hunk 7]
```diff
@@ -78,29 +74,31 @@ def stackup_layers(self):
...
+        -------
+        dict
+            Dictionary of materials.
+        """
         mats = {}
         for el in self.parent.edbutils.MaterialSetupInfo.GetFromLayout(self.parent.active_layout):
             mats[el.Name] = el
         return mats
 
+
```
Comment (PipKat) [inline pyaedt/edb_core/stackup.py:101]:
```suggestion
```

**Pull request merged**